fe1-web/ingestion/Watcher.ts

```diff
@@ -3,7 +3,7 @@ import {
   getLaoMessagesState, getLaosState, processMessages,
 } from 'store';
 import { ExtendedMessage } from 'model/network/method/message';
-import { handleMessage } from './handlers/Default';
+import { messageRegistry } from 'App';
 
 export function makeMessageStoreWatcher(store: Store) {
```

Comment:
this will, unfortunately, create an import loop again, as `App` is the entrypoint and therefore imports (indirectly) the whole application.

We may need to have something like

```suggestion
export function makeMessageStoreWatcher(store: Store, messageRegistry: MessageRegistry) {
```